static/js/components/dashboard/CourseRow.js

```diff
@@ -47,9 +47,7 @@ export default class CourseRow extends React.Component {
   );
 
   hasPaidForAnyCourseRun = (course: Course): boolean => (
-    (course.runs.length > 0) ?
-      R.any(R.propEq('has_paid', true), course.runs) :
-      false
+    R.any(R.propEq('has_paid', true), course.runs) || false
```

Comment:
I don't think we need `|| false` here, unless I'm missing something - if any elements in `course.runs` fail the predicate function (`R.propEq('has_paid', true)`) or if `course.runs` is `[]`, then we'll be doing `false || false // -> false`, and in cases where one of the elements passes we'll be doing `true || false // -> true`.